src/bench/coin_selection.cpp

```diff
@@ -3,25 +3,19 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <bench/bench.h>
-#include <wallet/wallet.h>
```

Comment:
14822 is somewhat partial since we don't have `test/sanitizer_suppressions/lsan` yet (you could maybe bckport the PR that added it in this PR?)

Reply:
Merge bitcoin#14794: tests: Add AddressSanitizer (ASan) Travis build (currently staged with Kitty)

Reply:
IMO, let's either mark 14822 as partial or backport 14794